### PR TITLE
Cosmos Yml pipeline configuration

### PR DESCRIPTION
**Pull request by g2vinay**

### CODE REVIEW

File: sdk/cosmos/tests.live.yml [hunk 1]
```diff
@@ -0,0 +1,429 @@
+trigger:
```
Comment (weshaggard) [inline sdk/cosmos/tests.live.yml]: You want to name this one tests.yml and your other one ci.yml to match our conventions.

Comment (g2vinay) [reply sdk/cosmos/tests.live.yml]: done

File: sdk/cosmos/tests.yml [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+
+    timeoutInMinutes: 240
+
+    strategy:
+      maxParallel: 1
+      matrix:
+        Session_Single_Region_ReadMyWrites:
+          OSVmImage: 'vs2017-win2016'
+          CosmosAccountHost: $(SINGLE_SESSION_ONE_ACCOUNT_HOST)
+          CosmosAccountKey: $(SINGLE_SESSION_ONE_ACCOUNT_KEY)
```
Comment (danieljurek) [inline sdk/cosmos/tests.yml]: Secrets should be stored in accordance with our testing guidelines.

Comment (g2vinay) [reply sdk/cosmos/tests.yml]: updated to use variable groups.

**Pull request merged**